src/common/config_parser.cpp

```diff
@@ -508,6 +508,18 @@ void ConfigParser::addOptionsTraining(cli::CLIWrapper& cli) {
      "Overlap model computations with MPI communication",
      true);
 
+  // model quantization training
```

Comment:
Since this is an isolated group of options, could you extract it to a subroutine like other our `addSuboptions*`?